### DISCUSSION TITLE
idea add more fuel rods

### DISCUSSION THREAD

**Discussion by litelmedox** [HbmMods/Hbm-s-Nuclear-Tech-GIT#640] [General]
so whe can have more choise

**Comment by Nos74** [comment]
no

**Reply by litelmedox** [reply]
why

**Reply by Shibva** [reply]
im going to say this now, you don't wanna go down that road. look where I got to as a result.

take this warning, from a damned soul

**Reply by Vaern** [reply]
you literally made a suggestion an hour ago
